src/setup/templates/cortx_rgw.conf

```diff
@@ -0,0 +1,14 @@
+[global]
```

Comment:
Pls move this file to "conf/cortx_rgw.conf" instead of "src/". Common convention followed.

Reply:
done